dashboards-observability/public/components/visualizations/charts/data_table/data_table.tsx

```diff
@@ -46,6 +33,30 @@ export const DataTable = ({ visualizations, layout, config }: any) => {
     metadata: { fields = [] },
   } = visualizations.data.rawVizData;
 
+  const { dataConfig = {} } = visualizations?.data?.userConfigs;
+  const enablePagination =
+    typeof dataConfig?.chartStyles?.enablePagination !== 'undefined'
+      ? dataConfig?.chartStyles?.enablePagination
+      : visualizations.vis.enablepagination;
+
+  const showTableHeader =
+    typeof dataConfig?.chartStyles?.showTableHeader !== 'undefined'
+      ? dataConfig?.chartStyles?.showTableHeader
+      : visualizations.vis.showtableheader;
+
+  const colunmFilter =
+    typeof dataConfig?.chartStyles?.colunmFilter !== 'undefined'
+      ? dataConfig?.chartStyles?.colunmFilter
+      : visualizations.vis.colunmfilter;
+
+  const columnAlignment =
+    dataConfig?.chartStyles?.columnAlignment || visualizations.vis.columnalignment;
+
+  const columnWidth =
+    typeof dataConfig?.chartStyles?.columnWidth !== 'undefined'
+      ? dataConfig?.chartStyles?.columnWidth
+      : visualizations.vis.columnwidth;
```

Comment:
These two statements are doing the same thing, but in different ways. Line # 52 and 53 looks better. Lets stick to that.

Reply:
Hi @spattnaik , 

As columnWidth is a numerical type, So in case if the user will enter 0 value in the case default columnalignment will be assigned as 1st statement will be false, to avoid this case I have to check the type there.

Thanks!